_utilities/layout-grid.md

```diff
@@ -504,116 +357,150 @@ Add `grid-gap` to a grid row to add a gap (or gutter) between each column in the
-@include grid-container;
-@include grid-container($container-size);
-
-// Creates a wrapper for a series of columns
-@include grid-row;
-
-// Specify the width between columns
-// $gap-size can be sm, md, lg, 2px, 05, 1, 3, 4, 6
-@include grid-gap;
-@include grid-gap($gap-size);
-
-// Make the element full-width
-@include u-width(full);
-
-// Specify the number of columns the element should span
-// $columns can be auto, fill, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12
-@include grid-col;
-@include grid-col($columns);
-
-// Get fancy by offsetting or changing the display order
-// $offset can be none, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include grid-offset($offset);
-
-// $order can be first, last, initial, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11
-@include u-order($order);
-{% endhighlight %}
+Visit the [settings page]({{ site.baseurl }}/documentation/settings) for more information and available settings.
 
-### Example usage
-You can modify the variables to your custom values or just use the mixins with their default values. Here’s an example of using the default settings to create a two-column layout with a gap.
+<section id="utility-mixins" class="padding-top-4">
+  <h2 class="site-h2 margin-y-0">Utility mixins</h2>
+  {% include utilities/utility-mixin-intro.html %}
 
-{% highlight scss %}
-.example-container {
-  @include grid-container;
-}
-
-.example-row {
-  @include grid-row;
-
-  // Add column gaps
-  &.content-row {
-    @include grid-gap;
-  }
-}
-
-.example-content-main {
-  @include u-width(full);
-
-  @include at-media(tablet) {
-    @include grid-col(6);
-  }
-
-  @include at-media(desktop) {
-    @include grid-col(8);
-  }
-}
+  <table class="usa-table--borderless site-table-responsive site-table-simple">
+    <thead>
+      <tr>
+        <th scope="col" class="tablet:maxw-card-lg display-inline-flex flex-2">Utility</th>
+        <th scope="col" class="display-inline-flex flex-2">Mixin</th>
+        <th scope="col" class="display-inline-flex flex-1">Accepted values</th>
+        <th scope="col" class="display-inline-flex flex-2">Example</th>
+      </tr>
+    </thead>
+    <tbody class="font-mono-2xs">
+      <tr>
+        <td data-title="Utility" class="tablet:maxw-card-lg flex-2">
+          <span>
+            .grid-container <br />
+            .grid-container-<code>container size</code>
+          </span>
+        </td>
+        <td data-title="Mixin" class="display-inline-flex flex-2">
+          <span>
+            grid-container<code>(container size)</code>
+          </span>
+        </td>
+        <td data-title="Accepted values" class="display-inline-flex flex-1">
+          <span>
+            mobile, mobile-lg, tablet, tablet-lg, desktop, desktop-lg, or widescreen
+          </span>
+        </td>
+        <td data-title="Example" class="display-inline-flex flex-2">
+          <span>
+            @include grid-container <br/>
+            @include grid-container("tablet")
```

Comment:
Do we want to do this for all utility mixins? I hadn't included this originally because that wasn't the existing pattern throughout sites utility mixin guidance.

Here's [display](https://designsystem.digital.gov/utilities/display/#utility-mixins-2) mixins for example:

![image](https://github.com/user-attachments/assets/e1de3c41-b154-47ab-8ee3-2ca410c1964d)

Reply:
I like the idea of including semi-colons on this page _in particular_ because we offer multiple `@include` examples in the same table cell. Adding semi-colons would:
1. Follow expected code convention
2. Clearly separate distinct `@include` statements in the same table cell

Unresolving this comment and added some code suggestions in my comments below.

Reply:
That makes sense to me! I considered adding it to only the lines with multiple examples but kept em all to be consistent.

Resolved in [c892bf4](https://github.com/uswds/uswds-site/pull/2276/commits/c892bf454ffbb136fab1f4fa1fa493265109aff9)